_overviews/toolkit/munit-clues.md

```diff
@@ -9,8 +9,135 @@ next-page: munit-asynchronous-tests
 
 {% include markdown.html path="_markdown/install-munit.md" %}
 
-<!-- 
-Tutorial about clues.
 
-See https://scalameta.org/munit/docs/assertions.html#assert
- -->
\ No newline at end of file
+In MUnit tests outputs are presented to the user in different ways depending on used assertion.
+For example `assertEquals` failure returns not only stack trace, 
+but also by default includes reason of the failure: `values are not the same` along with differences between two values.
```

Comment:
The first sentences should be a summary of the tutorial.

Something along those lines:
```
In MUnit we can use `clues` to get better error reports. There are two kind of clues:
- the `clue` argument of the `assert` method to print an explanation of the error
- the `clue` method to choose which values to print in the error report
```

Then we can have two parts.

## Using the `clue` argument

## Using the `clue` method

Reply:
Optionally, there could also be a short introduction to the problem, to get the user to feel that we are speaking exactly about the problem that he came to us with. Like
```
Normally, the MUnit will only provide the `values are not the same` error message. It's often not enough for making readable test reports and we can use `clues` to get better error reports. There are two kind of clues:
- the `clue` argument of the `assert` method to print an explanation of the error
- the `clue` method to choose which values to print in the error report
```